pkg/api/apiloader.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"io/ioutil"
 
+	"github.com/Azure/acs-engine/pkg/agentPoolOnlyApi/v20170831"
```

Comment:
Yes, this is just temporarily placement (and probably will remain so for 2-3 weeks or so) to not disrupt/confuse with the prod changes already in place.